doc/BENCHMARKING.md

```diff
@@ -13,6 +13,8 @@ In general, we run each IO operation for 30 seconds against a 100 GiB file. But
 
 ***readdir workload*** - we measure how long it takes to run `ls` command against directories with different size. Each directory has no subdirectory and contains a specific number of files, range from 100 to 100000 files, which we have to create manually using fio then upload them to S3 bucket before running the benchmark. The fio configuration files for creating them can be found at path [mountpoint-s3/scripts/fio/create/](../mountpoint-s3/scripts/fio/create).
 
+***write workload*** - we measure write throughput by using [dd](https://man7.org/linux/man-pages/man1/dd.1.html) command to simulate sequential write workloads.
```

Comment:
We plan to move to `fio` in the future, right? Can you note that here and then, for posterity, also add a comment on the PR about _why_ we're using `dd` for now?

Reply:
Yes, that's right. updated the doc.